web/client/src/components/request/ReqModule.jsx

```diff
@@ -0,0 +1,269 @@
+import React, { useState } from "react";
+import { Formik, Form, Field, FieldArray } from "formik";
+import {
+  Select,
+  CircularProgress,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  TextField,
+  Typography,
+  Button,
+  Grid,
+  OutlinedInput,
+  Dialog,
+  DialogTitle,
+  DialogActions,
+  DialogContent,
+  FormHelperText,
+  IconButton,
+} from "@material-ui/core";
+import AddCircleOutlineIcon from "@material-ui/icons/AddCircleOutline";
+import DeleteIcon from "@material-ui/icons/Delete";
+import "../../styles/request.css";
+import { object, string } from "yup";
+
+/**
+ * @return {Formik} component using Formik for creating send Request
+ */
+export default function ReqModule() {
+  const [open1, setOpen1] = useState(false);
+  const [open2, setOpen2] = useState(false);
```

Comment:
You could change the names of these variables so they represent the dialog section they are opening just to make it more clear what it is being used for.

Reply:
Fixed, but idk why you can't see the commit on the PR

Reply:
Apparently github is having issues, https://www.githubstatus.com/

Reply:
Github is having some issues,https://www.githubstatus.com/

Reply:
Seconded, these variable names should be changed